packages/web/src/components/edit/fields/price-and-audience/PriceAndAudienceMenuFields.tsx

```diff
@@ -159,7 +152,7 @@ export const PriceAndAudienceMenuFields = (
             disabled={isPublishDisabled}
           />
         )}
-        {isUsdcUploadEnabled ? (
```

Comment:
nit: can probably remove `isUsdcUploadEnabled` as well but can save for another PR

Reply:
ill do it here, good call